plugins/newsdownloader.koplugin/main.lua

```diff
@@ -245,11 +246,15 @@ function NewsDownloader:processAtom(feeds, limit)
         if limit ~= 0 and index - 1 == limit then
             break
         end
-        self:downloadFeed(feed, feed_output_dir)
+        if download_full_article then
```

Comment:
The logic is now reversed due to the variable name change, so `if not download_full_article` ;-)

Reply:
changed order

Reply:
Sure. 👍